pkg/apis/go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/eks-operator v1.0.1
 	github.com/rancher/norman v0.0.0-20200930000340-693d65aaffe3
-	github.com/rancher/rke v1.2.0
+	github.com/rancher/rke v1.2.2-rc1
```

Comment:
No go.sum changes ?

Reply:
there are go.sum changes too=> https://github.com/rancher/rancher/pull/29578/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63R1029

Reply:
I mean in `pkg/apis/`

Reply:
ah, thanks!